src/Yoti.Auth/BaseAttribute.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Yoti.Auth.Anchors;
+
+namespace Yoti.Auth
+{
+    /// <summary>
+    /// A class to represent a base Yoti attribute, without any generics. A Yoti attribute
+    /// consists of the attribute name, an associated <see cref="YotiAttributeValue"/>,
+    /// and a list of <see cref="Anchor"/>s from Yoti. It may hold one or more anchors,
+    /// which specify how an attribute has been provided and how it has been verified
+    /// within the Yoti platform.
+    /// </summary>
+    public class BaseAttribute
```

Comment:
1. Should it be public?
2. Should it be abstract?

Reply:
1. Since the derived class YotiAttribute is public, we can't change BaseAttribute without also changing YotiAttribute, as it would be less accessible.
1. Yes, good idea